### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by nunogois**
Small README fixes and update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +26,13 @@ Options:
           Port to listen for https connection on (will use the interfaces already defined) [env: TLS_SERVER_PORT=] [default: 3043]
 ```
 
+All supported [modes](#modes) share common configuration variables.
```
Comment (sighphyre) [inline README.md:29]: I'm not sure what this means here. Different modes will definitely have different variables, some modes will share some stuff

File: server/src/cli.rs [hunk 2]
```diff
@@ -4,7 +4,9 @@ use clap::{ArgGroup, Args, Parser, Subcommand};
 
 #[derive(Subcommand, Debug, Clone)]
 pub enum EdgeMode {
+    /// Run in offline mode
```
Comment (sighphyre) [inline server/src/cli.rs:7]: I don't think these comments add much, they seem to be describing what the enum variant does. If the enum is unclear, I think we should rather just rename it

**Review by sighphyre — COMMENTED** [review]
Looks good! Left some points of discussion, feel free to tell me I'm wrong

File: server/src/cli.rs (hunk 2 above)
Comment (sighphyre) [reply server/src/cli.rs:7]: Fair enough!

File: README.md (hunk 1 above)
Comment (sighphyre) [reply README.md:29]: Oooohhh. Yeah sure, the original is not a very clear description to me. I think you're right - let's remove it and put it back in when we actually have that section stabilised. Also okay with just leaving it. Gonna slap an approve on this and leave it up to you

**Review by sighphyre — APPROVED** [review]
G

**Review by chriswk — APPROVED** [review]
LGTM as well. Simon brought up some valid points, but I see you resolved those. More eyes on documentation is always good :)

**Pull request merged**